web/alerts.go

```diff
@@ -50,6 +50,8 @@ type AlertsHandler struct {
 	RuleManager manager.RuleManager
 
 	mutex sync.Mutex
+
+	PathPrefix string
```

Comment:
The empty lines here are getting weird :) Let's get rid of them completely and put the mutex at the very top.